testing/src/main/java/org/oppia/android/testing/OppiaTestRule.kt

```diff
@@ -15,6 +16,7 @@ class OppiaTestRule : TestRule {
         val currentPlatform = getCurrentPlatform()
         if (currentPlatform in targetPlatforms) {
           // Only run this test if it's targeting the current platform.
+          AccessibilityChecks.enable()
```

Comment:
@rt4914 why aren't we using 
`AccessibilityChecks.enable().setRunChecksFromRootView(true)`
For example if we enable a checkox which makes some other views visible, then I think we must be able to test the complete view  hierarchy

Reply:
Makes sense. I will use that. For now, I will need to close this PR.